framework/src/main/java/org/checkerframework/framework/qual/PostconditionAnnotation.java

```diff
@@ -33,10 +33,10 @@
  * <pre><code>
  * {@literal @}PostconditionAnnotation(qualifier = MinLen.class)
  * {@literal @}Target({ElementType.METHOD, ElementType.CONSTRUCTOR})
- * public {@literal @}interface EnsuresMinLen {
- *   String[] value();
- *   {@literal @}QualifierArgument("value")
- *   int targetValue() default 0;
+ *  public {@literal @}interface EnsuresMinLen {
+ *    String[] value();
+ *    {@literal @}QualifierArgument("value")
+ *    int targetValue() default 0;
```

Comment:
Extraneous whitespace changes?

Reply:
The whitespace was wrong before; this makes it line up in the generated HTML.  I could move it into a different pull request if you prefer.